Library/Homebrew/utils/spdx.rb

```diff
@@ -5,19 +5,111 @@
 module SPDX
   module_function
 
-  JSON_PATH = (HOMEBREW_LIBRARY_PATH/"data/spdx.json").freeze
+  JSON_PATH = (HOMEBREW_LIBRARY_PATH/"data").freeze
   API_URL = "https://api.github.com/repos/spdx/license-list-data/releases/latest"
 
-  def spdx_data
-    @spdx_data ||= JSON.parse(JSON_PATH.read)
+  def license_data
+    @license_data ||= JSON.parse (JSON_PATH/"spdx_licenses.json").read
+  end
+
+  def exception_data
+    @exception_data ||= JSON.parse (JSON_PATH/"spdx_exceptions.json").read
   end
 
   def latest_tag
     @latest_tag ||= GitHub.open_api(API_URL)["tag_name"]
   end
 
   def download_latest_license_data!(to: JSON_PATH)
-    data_url = "https://raw.githubusercontent.com/spdx/license-list-data/#{latest_tag}/json/licenses.json"
-    curl_download(data_url, to: to, partial: false)
+    data_url = "https://raw.githubusercontent.com/spdx/license-list-data/#{latest_tag}/json/"
+    curl_download(data_url + "licenses.json", to: to/"spdx_licenses.json", partial: false)
+    curl_download(data_url + "exceptions.json", to: to/"spdx_exceptions.json", partial: false)
+  end
+
+  def parse_license_expression(license_expression)
+    licenses = []
+    exceptions = []
+
+    case license_expression
+    when String, Symbol
+      licenses.push license_expression
+    when Hash
+      key = license_expression.keys.first
```

Comment:
What happens to the non-first key?

Reply:
Hmm, looks like it's ignored now.

This would happen in cases like this:

```ruby
license any_of: [
  all_of: ["MIT", "0BSD"],
  "0BSD" => { with: "LLVM-exception" },
]
```

I'll try to fix it later tonight.